### ISSUE TITLE
Editing a row in a table view sometimes results on white-on-white text

### ISSUE THREAD

**Issue by wjk** [ProfileCreator/ProfileCreator#70]
See below screenshot for an example.

<img width="913" alt="screenshot of ProfileCreator" src="https://user-images.githubusercontent.com/1915951/45328280-458daf00-b529-11e8-8a35-f1ca5c0dfc88.png">

**Comment by erikberglund** [comment]
Thanks, yes I noticed that yesterday too. I think it's related to #66 when I remade the wrong input highlighting and changed a setting for almost all textfields. I will try to find what causes that.

**Comment by erikberglund** [comment]
I am still seeing this but much more rare after the fix I added, so it might not be completely fixed in the next update. I will continue to investigate this.

**Comment by roodavis** [comment]
I was modifying payload Parental Controls: Application Controls, after dragging and dropping Google Chrome into the Whitelisted Applications by Code Signature window I noticed it was the only one that had a parentheses "(" in the Code requirement field. When clicking in this field to remove the parentheses the text in this field is not visible, possibly a font color issue as the cursor moves through the field with the keyboard arrows as expected. I came here and saw this was already listed as an issue at that point and then went back to ProfileCreator. When I used the up arrow key to go to the beginning of the field I then used the right arrow key to go past the parentheses then deleted the character and the rest of the text appeared, but I could not use the down arrow key to go to the end of the string and delete the closing parentheses, it only took me to the end of the visible field.

I then removed another character then clicked Undo so the deleted character returned but it was then only visible when the entry was highlighted.

**Comment by erikberglund** [comment]
Yeah I have seen this issue as well when using drag and drop and I'll look at it. As well as the issue that one cant move around in the textfield.

But @roodavis you should never edit the codesignature part when doing importing, the parenthesis have to be there for the signature to be correct as it's grouping different things when checking the requirement. For chrome it looks like this: ```(identifier "com.google.Chrome" or identifier "com.google.Chrome.beta" or identifier "com.google.Chrome.dev" or identifier "com.google.Chrome.canary") and (certificate leaf = H"85cee8254216185620ddc8851c7a9fc4dfe120ef" or certificate leaf = H"c9a99324ca3fcb23dbcc36bd5fd4f9753305130a")``` so if you modify anything here this will not verify correctly.

**Comment by wjk** [comment]
This appears to no longer happen as of ProfileCreator 0.3.7.
